crates/common/consensus/lean/src/block.rs

```diff
@@ -70,5 +71,5 @@ impl From<Block> for BlockHeader {
     Debug, Default, PartialEq, Eq, Clone, Serialize, Deserialize, Encode, Decode, TreeHash,
 )]
 pub struct BlockBody {
-    pub votes: VariableList<Vote, U4096>,
+    pub attestations: VariableList<Arc<SignedVote>, U4096>,
```

Comment:
```suggestion
    pub attestations: VariableList<SignedVote, U4096>,
```

I think this part doesn't need `Arc` as well.